src/libstd/yk_swt.rs

```diff
@@ -15,5 +54,23 @@
 #[cfg_attr(not(stage0), no_trace)]
 #[cfg(not(test))]
 fn rec_loc(crate_hash: u64, def_idx: u32, bb_idx: u32) {
-    // Not implemented.
+    TRACE.with(|rc| {
+        let mut trace_o = rc.borrow_mut();
+        match trace_o.as_mut() {
+            Some(trace) => trace.push(MirLoc{crate_hash, def_idx, bb_idx}),
+            None => (), // Tracing is disabled, do nothing.
```

Comment:
"Disabled" or "not active"?

Reply:
Both I guess. How about:

"We are not currently tracing, do nothing"

Reply:
Agreed.

Reply:
We had resolved the discussion (and I trust you to do what you'd suggested you would do). No point having things hanging around in such a case I think?

Reply:
Ah I see. This isn't quite what I thought. The problem with this strategy is that I have to go unfolding threads to see if I've addressed them or not. I thought you would only resolve them when you saw the commit that addresses them.

But that might also be sub-optimal?